src/envo/envo-edit.owl

```diff
@@ -16314,18 +16314,28 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00004683> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00005738> (foam)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_00005738> "https://en.wikipedia.org/wiki/Foam"^^xsd:string)
```

Comment:
In this case, you've changed the semantics of the term quite a bit - you should either: 

### Option 1. 
i) Keep this term as an environmental material called "foamy environmental material", but defined as "An environmental material which has the quality of a foam". This would be similar to liquid environmental material etc

ii) In addition, create a new term that's the "quality of a foam" in the PATO hierarchy (sibling to quality of a liquid etc)

You can then add sea foam underneath it.

### Option 2

Obsolete ENVO_00005738 and create a new term that's only the quality